scripts/new_project.kts

```diff
@@ -46,6 +48,22 @@ object NewProject {
             return@copyRecursively OnErrorAction.TERMINATE
         }
     }
+
+    private fun cleanNewProjectFolder() {
+        executeCommand("./../$appNameWithoutSpace/gradlew -p ../$appNameWithoutSpace clean")
+        executeCommand("./../$appNameWithoutSpace/gradlew -p ../$appNameWithoutSpace/buildSrc clean")
+        File("../$appNameWithoutSpace/.idea")?.deleteRecursively()
+        File("../$appNameWithoutSpace/.gradle")?.deleteRecursively()
+        File("../$appNameWithoutSpace/buildSrc/.gradle")?.deleteRecursively()
+        File("../$appNameWithoutSpace/.git")?.deleteRecursively()
```

Comment:
relative-path like `./../` is error-prone and it may cause issues if we update the project path in the future 💭  (we have to find all mentioned paths and update them 🤪 )
I would suggest getting the current working directory + the new project directory instead. What do you think?

Reply:
@manh-t Fair point 👍 

Could you show us some example code for better understanding, please?

Reply:
@Tuubz @kaungkhantsoe currently the new project will be stored in the root `android-templates`. We can define the project path and then append the directories:
```
val root = File("").absolutePath.toString().replace(File.separator + "scripts", "")
val projectPath = root + File.separator + appNameWithoutSpace
listOf(".idea", ".gradle", "buildSrc" + File.separator + ".gradle", ".git").forEach {
    val targetFile = File(projectPath + File.separator + it)
    if (targetFile.exists()) targetFile.deleteRecursively()
}
```
Notice that the template can be init from any platform, so `File.separator` is recommended instead of `"/"` 😁

Reply:
> Notice that the template can be init from any platform, so File.separator is recommended instead of "/" 😁

@manh-t Just curious, any idea which platform doesn't use `"/"` as a file separtor?

Reply:
@Tuubz windows for sure `"\"`

Reply:
Aaaah, I see. Thanks 😁 

Yes, let's do it with the `File.separator` then 🚀

Reply:
Updated 🚀